hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -169,6 +161,27 @@
         startCleanupThread();
     }
 
+    private OperationExecutor initOperationExecutor() {
+        return new ProgressiveOperationExecutor(
```

Comment:
For the time being it is enabled by default so I can run all the unit tests and see if something breaks. Will be reverted when I make the final PR.